package.json

```diff
@@ -24,30 +27,23 @@
     "url": "https://github.com/yamadapc/js-written-number"
   },
   "devDependencies": {
-    "blanket": "1.1.7",
-    "grunt": "^1.0.3",
-    "grunt-browserify": "^5.3.0",
-    "grunt-cli": "^1.2.0",
-    "grunt-contrib-clean": "^1.1.0",
-    "grunt-contrib-uglify": "^2.3.0",
+    "@babel/core": "^7.5.5",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/register": "^7.5.5",
+    "babel-eslint": "^10.0.2",
+    "babel-loader": "^8.0.6",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.0.3",
+    "eslint": "^6.0.1",
+    "eslint-config-airbnb-base": "^13.2.0",
+    "eslint-plugin-import": "^2.18.2",
     "mocha": "^3.5.3",
```

Comment:
Since we are dropping support node versions less than 8, can you please verify 
- we are using latest versions for devDependencies
- remove unused dependencies from package.json (may be mocha ?)